lib/ohai/plugins/linux/platform.rb

```diff
@@ -81,6 +81,14 @@ def os_release_file_is_cisco?
     file_exist?("/etc/os-release") && os_release_info["CISCO_RELEASE_INFO"]
   end
 
+  #
+  # If /etc/os-release indicates we are CentOS Stream
+  #
+  # @returns [Boolean] if we are CentOS Stream according to /etc/os-release
+  def os_release_file_is_centos_stream?
+    file_exist?("/etc/os-release") && os_release_info["NAME"].match?("Stream")
```

Comment:
I'd recommend specifically testing for `CentOS Stream` here in case something else comes up in the future with `Steam` in the name.